unledger/main.go

```diff
@@ -0,0 +1,223 @@
+package main
+
+import (
+	"bytes"
+	"encoding/pem"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
```

Comment:
I guess we can use whatever if we assume this will be on airgapped machines only. but does anyone want to vouch for this package?